package.json

```diff
@@ -64,15 +64,22 @@
   },
   "devDependencies": {
     "@edx/browserslist-config": "^1.2.0",
-    "@edx/frontend-build": "13.0.1",
     "@edx/reactifex": "^2.1.1",
+    "@openedx/frontend-build": "13.0.27",
     "@testing-library/jest-dom": "5.16.5",
     "@testing-library/react": "12.1.5",
     "axios-mock-adapter": "1.21.2",
     "glob": "7.2.3",
     "husky": "7.0.4",
     "jest": "27.5.1",
+    "postcss-loader": "^8.0.0",
     "prettier": "2.8.1",
-    "rosie": "2.1.0"
+    "react-intl": "^6.6.1",
+    "rosie": "2.1.0",
+    "sass": "^1.70.0",
+    "sass-loader": "^14.0.0",
+    "sharp": "^0.33.2",
+    "source-map-loader": "^5.0.0",
+    "style-loader": "^3.3.4"
```

Comment:
It's unclear to me why these aren't being found anymore when installed as part of frontend-build I'll continue to investigate.